asf/sam0/include/samr35/README

```diff
@@ -37,3 +37,4 @@ License Link:
 
 Patch List:
    * Fix anonymous bit-fields with qualifiers.
+   * Fix DAC rename EMPTY bit-fields.
```

Comment:
this is not a fix, it is a rename.

```diff
-* Fix DAC rename EMPTY bit-fields.
+* Rename DAC EMPTY bit-fields to DBEMPTY to avoid collision with Zephyr macros.
```